src/cmd/linuxkit/pkglib/pkglib.go

```diff
@@ -169,6 +178,36 @@ func NewFromCLI(fs *flag.FlagSet, args ...string) (Pkg, error) {
 		}
 	})
 
+	var srcHashes string
+	sources := []pkgSource{pkgSource{src: pkgPath, dst: "/"}}
```

Comment:
I think `gofmt` is going to want to make this  `[]pkgSource{{src: pkgPath, dst: "/"}}` which is more concise (the inner type is implicit)